sdcm/gemini_thread.py

```diff
@@ -96,20 +95,27 @@ def __init__(self, test_cluster: BaseCluster | BaseScyllaCluster, oracle_cluster
             "max-clustering-keys": 4,
             "min-clustering-keys": 2,
             "partition-key-distribution": "uniform",  # Distribution for hitting the partition
-            # These two are used to control the memory usage of Gemini
-            "token-range-slices": 10000,  # Number of partitions
-            "partition-key-buffer-reuse-size": 128,  # Internal Channel Size per partition value generation
+            "token-range-slices": 10000,
```

Comment:
why not renaming to something like "partitions-number"?

Reply:
not related to this PR